web/html/src/manager/admin/config/use-monitoring-api.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 import {useState} from 'react';
-import * as Network from 'utils/network';
+import Network from 'utils/network';
 
 import type {JsonResult} from "../../../utils/network";
```

Comment:
Now that we are on it, I wonder if with some good `regex` it is possible to:
- find and replace all the `"....."` to `'.....'`
- keep a space between brackets from `{abc}` to `{ abc }`

while using imports. I see we are standardizing it so it would be nice, but not really required.

Reply:
This disturbs me as well :rofl:

On the other hand, I prefer single quotes and `{abc}` without spaces, but I guess we can leave it to @Etheryte to decide

Reply:
This caught my eye as well, but I didn't want to throw everything along with the kitchen sink into one PR. I'll create a follow-up card and do it there if that works for you?

Reply:
If I could vote, I would:

- 2 space only, I think we already have it in place as a *standardization*, probably we don't have it enforced nor a checkstyle rule to blame who breaks it.
- single quotes
- no real preference for space or no-space on `{abc}`

Time for a strong check style test to fail if someone breaks them, whatever the rules will be? We do in Java, why not here?

Reply:
I created https://github.com/SUSE/spacewalk/issues/13437 to track this with a short summary of the above.